### PR TITLE
Add double holsters

### PR DESCRIPTION
**Pull request by Holli-Git**
#### Summary
Content "Double Decker Holsters"

#### Purpose of change

Wanted to be able to carry more guns that were easy access
#### Describe the solution

Adds two double holsters, one that has each holster side by side, and the other with two holsters front and back.
#### Describe alternatives you've considered

Having a belt which can have multiple holsters attached to it.
#### Testing

Tested locally, works fine.

### CODE REVIEW

**Comment by GuardianDll** [comment]
Why not add it into loot groups to spawn in game?

**Comment by Paprika-H** [comment]
> Why not add it into loot groups to spawn in game?

I cannot find any real world examples of this stuff, so it's a survivor craft exclusive

File: data/json/items/armor/holster.json [hunk 1]
```diff
@@ -455,6 +455,102 @@
     "flags": [ "SKINTIGHT", "WATER_FRIENDLY" ],
     "armor": [ { "encumbrance": 0, "coverage": 50, "covers": [ "torso" ], "specifically_covers": [ "torso_lower" ] } ]
   },
+  {
+    "id": "side_double_holster",
+    "type": "ARMOR",
+    "name": { "str": "side by side double holster" },
```
Comment (TheShadowFerret) [inline data/json/items/armor/holster.json:461]:
```suggestion
    "name": { "str": "side-by-side double holster" },
```

File: data/json/items/armor/holster.json [hunk 2]
```diff
@@ -455,6 +455,102 @@
...
+        "coverage": 5,
+        "covers": [ "leg_l", "leg_r" ],
+        "specifically_covers": [ "leg_hip_r", "leg_hip_l" ]
+      }
+    ]
+  },
+  {
+    "id": "FB_double_holster",
+    "type": "ARMOR",
+    "name": { "str": "front to back double holster" },
```
Comment (TheShadowFerret) [inline data/json/items/armor/holster.json:509]:
```suggestion
    "name": { "str": "front-to-back double holster" },
```

**Review by TheShadowFerret — CHANGES_REQUESTED** [review]

**Comment by Termineitor244** [comment]
> > Why not add it into loot groups to spawn in game?
> 
> I cannot find any real world examples of this stuff, so it's a survivor craft exclusive

You could always add it to the survivor zombies then.

**Pull request merged**